integration_tests/mockApis/curiousApi.ts

```diff
@@ -28,18 +28,21 @@ export default {
       },
     })
   },
-  stubGetLearnerEducation: (prisonerNumber: string) => {
+  stubGetLearnerEducation: (prisonerNumber: string, page = 0) => {
     return stubFor({
       request: {
         method: 'GET',
-        urlPattern: `/curiousApi/learnerEducation/${prisonerNumber}`,
+        urlPathPattern: `/curiousApi/learnerEducation/${prisonerNumber}`,
+        queryParameters: {
+          page: { equalTo: `${page}` },
+        },
       },
       response: {
         status: 200,
         headers: {
           'Content-Type': 'application/json;charset=UTF-8',
         },
-        jsonBody: learnerEducation,
+        jsonBody: learnerEducationPagedResponse(prisonerNumber),
```

Comment:
See my changes below to the `learnerEducation.ts` local mock data file, but the terminology was wrong previously.
A "learner education" refers to a single course or qualification. The response from the API however is a paged set, containing several learner education records.